src/scripts/main.js

```diff
@@ -1,8 +1,26 @@
 'use strict';
 
-const people = require('./lib/people');
+const people = require('./lib/people.json');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach(human => {
+  const { sex, born, died } = human;
+  const gender = (personSex) => personSex === 'm' ? 'Male' : 'Famele';
```

Comment:
Typo in the ternary operator for gender. Replace 'Famele' with 'Female'.

Reply:
done